webpage/index.html

```diff
@@ -1,45 +1,95 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <meta charset="UTF-8">
-  <title>Test</title>
-  <script type="text/javascript">
-
-    function LigthSwitch(){
-      alert("Button");
-    }
-    
-    function loadFile(filePath) {
-      var result = null;
-      var xmlhttp = new XMLHttpRequest();
-      xmlhttp.open("GET", filePath, false);
-      xmlhttp.send();
-      if (xmlhttp.status==200) {
-        result = xmlhttp.responseText;
-      }
-      return result;
-    }
-    
-    let interval = setInterval(ServerCheck, 500);
-    function ServerCheck() {
-        var temperature = loadFile("temperature.txt")
-        var ledStatus = loadFile("ledStatus.txt")
-        document.getElementById("Temperature").value = parseInt(temperature, 10);
-
-        if(parseInt(ledStatus, 10) == 1)
-            document.getElementById("Led").innerHTML="Led is now On";
-        else
-            document.getElementById("Led").innerHTML="Led is now Off";
-    }
-  </script>
+    <meta charset="UTF-8">
+    <title>Test</title>
+    <link rel="stylesheet" href="css/zeroing.css">
+    <link rel="stylesheet" href="css/style.css">
+    <script type="text/javascript">
+        function LigthSwitch() {
+            alert("Button");
+        }
+
+        function loadFile(filePath) {
+            var result = null;
+            var xmlhttp = new XMLHttpRequest();
+            xmlhttp.open("GET", filePath, false);
+            xmlhttp.send();
+            if (xmlhttp.status == 200) {
+                result = xmlhttp.responseText;
+            }
+            return result;
+        }
+
+        let interval = setInterval(ServerCheck, 500);
+
+        function updateTemperature(temperature) {
+            document.getElementById("Temperature").value = parseInt(temperature, 10);
+        }
+
+        function updateLed(ledStatus) {
+            if (parseInt(ledStatus, 10) == 1)
+                document.getElementById("ledON").hidden = true;
+            else
+                document.getElementById("ledON").hidden = false;
+        }
+
+        function ServerCheck() {
+            var temperature = loadFile("temperature.txt")
+            var ledStatus = loadFile("ledStatus.txt")
+
+            updateTemperature(temperature);
+            updateLed(ledStatus);
+
+        }
+
+        let counter = true;
+
+        function ChangeStatus() {
+            if (counter == 0) {

```

Comment:
I think this if statement is not necessary. You can just toggle booleans.
```suggestion
    var butOn = document.getElementById("butOn")
    var butOff = document.getElementById("butOff")
    butOn .hidden = !butOn .hidden;
    butOff .hidden = !butOff.hidden;
```